firmware/main/conductor/songs.ino

```diff
@@ -0,0 +1,126 @@
+/* 0 (C) */  N, N, N, N, N, N, N, N, N, N, N, N, N, T, N, N, 
+}
+
+staff_data_t pokemonLevelUp[SONG_ELEMENTS] = {
+/*           0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15 */
+/* 6 (B) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 5 (A) */  N, N, N, N, N, N, N, N, G, G, G, N, N, N, N, N,
+/* 4 (G) */  G, N, N, N, G, G, G, N, N, N, N, N, N, G, G, G,
+/* 3 (F) */  N, G, N, G, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 2 (E) */  N, N, G, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 1 (D) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 0 (C) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, 
+}
+
+staff_data_t funkyTown[SONG_ELEMENTS] = {
+/*           0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15 */
+/* 6 (B) */  N, N, N, N, N, N, N, N, N, N, F, N, N, N, N, N,
+/* 5 (A) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 4 (G) */  F, F, N, F, N, N, N, N, F, N, N, F, N, N, N, N,
+/* 3 (F) */  N, N, F, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 2 (E) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 1 (D) */  N, N, N, N, N, F, N, F, N, N, N, N, N, N, N, N,
+/* 0 (C) */  N, N, N, N, N, N, N, N, N, F, N, N, N, N, N, N, 
+}
+
+staff_data_t miiShopChannel[SONG_ELEMENTS] = {
+/*           0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15 */
+/* 6 (B) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 5 (A) */  N, N, P, N, N, P, N, N, N, N, N, N, N, N, N, N,
+/* 4 (G) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 3 (F) */  P, N, N, N, N, N, N, P, N, N, N, N, N, N, N, N,
+/* 2 (E) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 1 (D) */  N, N, N, P, N, N, N, N, P, P, P, N, N, N, N, N,
+/* 0 (C) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, 
+}
+
+staff_data_t somewhereOverTheRainbow[SONG_ELEMENTS] = {
+/*           0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15 */
+/* 6 (B) */  N, N, N, N, N, N, N, N, F, N, N, N, F, N, N, N,
+/* 5 (A) */  N, N, N, N, N, N, N, N, N, N, N, F, N, N, N, N,
+/* 4 (G) */  F, N, N, N, N, N, N, N, N, N, F, N, N, N, N, N,
+/* 3 (F) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 2 (E) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 1 (D) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 0 (C) */  N, N, N, N, F, N, N, N, N, N, N, N, N, N, F, N, 
+}
+
+staff_data_t jurassicPark[SONG_ELEMENTS] = {
+/*           0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15 */
+/* 6 (B) */  T, N, T, N, N, N, N, N, T, N, T, N, N, N, N, N,
+/* 5 (A) */  N, T, N, N, N, N, N, N, N, T, N, N, N, N, N, N,
+/* 4 (G) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 3 (F) */  N, N, N, N, T, N, N, N, N, N, N, N, T, N, N, N,
+/* 2 (E) */  N, N, N, N, N, N, T, N, N, N, N, N, N, N, T, N,
+/* 1 (D) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N,
+/* 0 (C) */  N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, N, 
+}
+
+show_song(jurassicPark);
+//show_song(somewhereOverTheRainbow);
+show_song(pokemonLevelUp);
+show_song(funkytown);
+show_song(miiShopChannel);
+show_song(axelf);
```

Comment:
what are these calls to show_song meant to do? I am not sure that they are even valid at this scope (outside of setup or loop)